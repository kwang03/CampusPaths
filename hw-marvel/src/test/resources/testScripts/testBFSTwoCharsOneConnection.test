#Tests BFS on a graph with two characters and one connection between them

LoadGraph myGraph twoCharsOnePath.csv

FindPath myGraph Char1 Char2

FindPath myGraph Char2 Char1