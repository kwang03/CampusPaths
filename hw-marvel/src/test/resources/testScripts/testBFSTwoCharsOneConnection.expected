#Tests BFS on a graph with two characters and one connection between them

loaded graph myGraph

path from Char1 to Char2:
Char1 to Char2 via Book1

path from Char2 to Char1:
Char2 to Char1 via Book1